src-built-in/adapters/indexedDBAdapter.js

```diff
@@ -134,6 +137,15 @@ const IndexedDBAdapter = function () {
 		return preface;
 	}
 
+	this.releaseQueue = () => {
+		if (this.queue.length) {
+			Logger.system.debug("IndexedDBAdapter.releaseQueue");
```

Comment:
It might be better to make this command always log with:
```Logger.system.debug(`IndexedDBAdapter.releaseQueue: ${this.queue.length} commands`);```
So you could tell that the queue was released, but there weren't any commands to process.

Reply:
I think clearCache is missing a queue push.